runtime/queries/column_timeseries.go

```diff
@@ -131,31 +139,35 @@ func (q *ColumnTimeseries) Resolve(ctx context.Context, rt *runtime.Runtime, ins
 		}()
 
 		rows, err := olap.Execute(ctx, &drivers.Statement{
-			Query:    `SELECT * from "` + temporaryTableName + `"`,
+			Query:    fmt.Sprintf(`SELECT %s as ts, * EXCLUDE(%s) FROM %s`, tsAlias, tsAlias, temporaryTableName),
 			Priority: priority,
 		})
 		if err != nil {
 			return err
 		}
 
-		results, err := convertRowsToTimeSeriesValues(rows, len(measures)+1, tsAlias)
+		results, err := rowsToData(rows)
+		meta := structTypeToMetricsViewColumn(rows.Schema)
 		rows.Close()
 		if err != nil {
 			return err
 		}
```

Comment:
Should `err` be checked before call to `structTypeToMetricsViewColumn`?

Reply:
Good catch.